modules/influxdb/src/main/java/org/testcontainers/containers/InfluxDBContainerV1.java

```diff
@@ -0,0 +1,165 @@
+package org.testcontainers.containers;
+
+import java.util.Collections;
+import java.util.Set;
+import org.influxdb.InfluxDB;
+import org.influxdb.InfluxDBFactory;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * See <a href="https://store.docker.com/images/influxdb">https://store.docker.com/images/influxdb</a>
+ */
+public class InfluxDBContainerV1<SELF extends InfluxDBContainerV1<SELF>> extends GenericContainer<SELF> {
+
+    public static final Integer INFLUXDB_PORT = 8086;
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("influxdb");
+    private static final String DEFAULT_TAG = "1.4.3";
+
+    private static final int NO_CONTENT_STATUS_CODE = 204;
+
+    @Deprecated
+    public static final String VERSION = DEFAULT_TAG;
+
+    private boolean authEnabled = true;
+    private String admin = "admin";
+    private String adminPassword = "password";
+
+    private String database;
+    private String username = "any";
+    private String password = "any";
+
+    /**
+     * @deprecated use {@link InfluxDBContainerV1(DockerImageName)} instead
+     */
+    @Deprecated
+    public InfluxDBContainerV1() {
+        this(DEFAULT_IMAGE_NAME.withTag(DEFAULT_TAG));
+    }
+
+    /**
+     * @deprecated use {@link InfluxDBContainerV1(DockerImageName)} instead
+     */
+    @Deprecated
+    public InfluxDBContainerV1(final String version) {
+        this(DEFAULT_IMAGE_NAME.withTag(version));
+    }
```

Comment:
Same here - consider making it a non-deprecated `String imageName` constructor

Reply:
So should I replace this constructor with `public InfluxDBContainerV1(final String imageName)`? This would cause the `InfluxDBContainer()` to fail. Same as the previous constructor you suggested to delete

Reply:
The reason we kept these constructors is to maintain binary compatibility.

Reply:
constructors are not inherited in Java - you need to explicitly define them again in `InfluxDBContainer`